nilearn/_utils/docs.py

```diff
@@ -513,17 +513,28 @@ def custom_function(vertices):
 # bg_on_data
 docdict['bg_on_data'] = """
 bg_on_data : :obj:`bool`, optional
-    If ``True``, and a ``bg_map`` is specified,
+    If ``True`` and a ``bg_map`` is specified,
     the ``surf_data`` data is multiplied by the background
-    image, so that e.g. sulcal depth is visible beneath
-    the ``surf_data``.
+    image, so that e.g. sulcal depth is jointly with ``surf_data``.
```

Comment:
I think we are missing a word in this sentence?

Reply:
You are right, I fixed that :)